software/controller/lib/hal/spi_chain.tpp

```diff
@@ -171,15 +174,19 @@ size_t DaisyChain<MaxSlaves, MaxRequestsPerSlave>::ParseProbeResponse(uint8_t *r
   }
 
   // If the response buffer is full of null commands, this most likely means the
-  // daisy chain is broken (more slaves than we expect)
-  if(num_slaves_==sizeof(response_buffer)) {
+  // daisy chain is broken (more slaves than we expect), stop sending any data.
```

Comment:
This smells like a creative way to fail :) Do we want an error counter for something like this? Or for anything else in these classes?